hack/determine_install_upgrade_version.py

```diff
@@ -0,0 +1,55 @@
+import yum
+import sys
+import os.path
+from version_utils import rpm
+
+try:
+	input_pkg = sys.argv[1]
+	input_pkg_obj = rpm.package(input_pkg)
+except IndexError:
+	sys.exit(1)
+except:
+	print "[ERROR] Incorrect package nevra format: " + input_pkg
+	sys.exit(1)
+
+input_pkg_name = str(input_pkg_obj.name)
+input_pkg_version_list = map(int, input_pkg_obj.version.split("."))
+search_version_list = input_pkg_version_list[:]
+search_version_list[1] = search_version_list[1] -1
+search_version = ".".join(map(str, search_version_list)[:2])
```

Comment:
If not, I think this could be written in such a way to be a little more clear:

```py
major, minor, *rest = input_pkg.version.split('.')
previous_version = '.'.join(major, minor+1, *rest)
```